app/controllers/search_controller.rb

```diff
@@ -2,8 +2,8 @@ class SearchController < ApplicationController
   def search
     session[:address] = params[:search] if params.has_key?(:search)
     @search = Listing.where(is_active: true)
-    @search = @search.where(address: session[:address])
+    @search = @search.near("#{session[:address]}", 5, order: 'distance')
```

Comment:
Is it necessary to put the `session[:address]` inside `"#{}"`?

Reply:
I tried using without the enclosing them inside double quotes, and it doesn't work,
I've noticed that in Geocoder's Github page, whenever using an address(string) the syntax needs it to be enclosed into double quotes,

given the session[:address] is a string of text and not a longitude and latitude
![image](https://user-images.githubusercontent.com/73863408/114979912-3bd93e80-9ebe-11eb-88ba-09bb19d959e2.png)